src/__test__/Reducers.test.js

```diff
@@ -0,0 +1,207 @@
+import userReducer from '../reducers/userReducer';
```

Comment:
Why don't we split our reducer tests to the different reducers instead of one big file?